demos/VM_Export/demoscript_Oct-VM-export.txt

```diff
@@ -0,0 +1,81 @@
+VM Export
+<Intro>
+In this demo we will be exporting a VM disk, and having a quick look at the behaviour of the process. 
+
+Why? We might use VM Export so that it can be imported into another cluster, platform, or system. 
+
+<Start>
+We start with a KubeVirt cluster. 
+This cluster has the virtctl client which includes additional commands to interact with VMs. 
+And we have a stopped Fedora 36 VM that we want to export.
+On the left we have our command pane. 
+On the top-right we are watching the VM export objects to see their phases.
+And on the lower-right we are watching the pods. 
+
+<0:15>
+To export the VM disk, we use virtctl with the 'vmexport create' option, and specify the name of the VM.
+And we see the VM export has been created and is pending as the pod is created.
+When the pod is running, we can see that the VM export is now ready.
+
+What does this mean?
+
+All PVCs, data volumes, and memorydump volumes on the VM will be exported.
+(No other volume types will be exported)
+Also, when you create a VM export, a dedicated export proxy server is created which can then be exposed to the outside world with an Ingress/Route or NodePort.
```

Comment:
Small detail `Also, when you create a VM export, a dedicated export proxy server is created which can then be exposed to the outside world with an Ingress/Route or NodePort.` is not technically incorrect, but the proxy is created when you enable the feature gate. The export server is created when you create a VM export, the proxy then talks to the export server and the route/ingress talks to the proxy. This eliminates the k8s API from the process.

Reply:
Thanks @awels - does this now accurately explain it?